src/fiskaltrust.Middleware.Interface.Grpc/GrpcPosFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using fiskaltrust.ifPOS.v1;
+using Grpc.Core;
+using ProtoBuf.Grpc.Client;
+using System;
+
+namespace fiskaltrust.Middleware.Interface.Grpc
+{
+    public class GrpcPosFactory
+    {
+        public IPOS CreatePosAsync(GrpcPosOptions options)
+        {
+            var uri = new Uri(options.Url);
+            var channel = new Channel(uri.Host, uri.Port, ChannelCredentials.Insecure);
```

Comment:
It doesn´t work yet, but should we make the `Secure` || `Insecure` Option available for the caller?

Reply:
Good idea, the ChannelCredentials can now be set in GrpcPosOptions